docs/Routes.md

```diff
@@ -34,8 +34,7 @@ fastify.route(options)
 They need to be in
   [JSON Schema](https://json-schema.org/) format, check [here](Validation-and-Serialization.md) for more info.
 
-  * `body`: validates the body of the request if it is a POST or a
-    PUT.
+  * `body`: validates the body of the request if it is a POST, PUT or PATCH method, and doesn't work with GET and HEAD methods.
```

Comment:
```suggestion
  * `body`: validates the body of the request if it is a POST, PUT, or PATCH method.
```
Second half of the sentence is extraneous, if a HTTP request method is not mentioned in the first half of the sentence then it should be assumed that it is not supported.